README.md

```diff
@@ -1,2 +1,16 @@
 # hw01
 Repository for homework 01
+
+# Introduction
+
+Hello. My name is Chandler James. I'm a second year PhD student in the political science department. Before graduate school, I worked as a staffer for a US congressman. My research interests include:
+
+* American politics
+* institutions 
+* public opinion
+
+
+You can follow me on *twitter* at this link -> http://twitter.com/chnjms
```

Comment:
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      X     |              |            |
| **Coding strategy**         |      X     |              |            |
| **Presentation**            |     X      |              |            |
| **Achievement, creativity** |           |       X       |            |
| **Ease of access**          |     X      |              |            |

Remarks:
*Great use of markdown elements!
*A bit short, could have gone on with more details to fully showcase the elements, but you did fulfill the rubric of the assignment.
*If you're interested you could link the URL in rich text, but not necessary.
*Great job!